diagnostic.md

```diff
@@ -71,13 +71,13 @@ Using Atom, open up `diagnostic.md` and write your answers below (where indicate
 If we are in the `project` directory and use `cd planning`, is a relative or absolute path being referenced? How do you know?
 
  <!-- Answer Starts Here -->
-
+relative, because we are not providing the full pathname.
  <!-- Answer Ends Here -->
 
 3. Now suppose that we have an image file living inside our project. Would we refer to it with an absolute or relative path? Why?
 
  <!-- Answer Starts Here -->
-
+absolute, so we can reference it regardless of where we are.
```

Comment:
In this case I would use a relative path since the image file lives inside our current working directory, project. As a result, an absolute path is not needed to access it.